LabelGenerator.py

```diff
@@ -683,6 +697,20 @@ def main() -> None:
         [9100000000,   9200000000,   3300000000],
     ]
 
+
+    # ############################################################################
+    # Resistor tolerance
+    # 1:    1%      brown
+    # 2:    2%      red
+    # 0.5:  0.5%    green
+    # 0.25: 0.25%   blue
+    # 0.1:  0.1%    violet
+    # 5:    5%      gold
+    # 10:   10%     silver
+    # ############################################################################
+
+    resistor_tolerance = 1 # in percentage
```

Comment:
Set to `None` by default, and use color `-3` for the tolerance stripe if `None`. This makes it backwards compatible with the current behavior.